docs/tutorials/want-to-use-python.md

```diff
@@ -274,6 +274,34 @@ Returns the hash value of this object, internally it calls the CPython's [`PyObj
 
 Returns a corresponding primitive value for this object, see [`Symbol.toPrimitive` on MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Symbol/toPrimitive) for more details.
 
+#### With ES-Modules
+> Requires Node.js >= `v12.11.1`
+
+use Node.js custom loader for better import-statement.
+``` js
+// app.mjs
+import { equal, ok } from 'assert';
+import { range, len } from 'py:builtins';
+import os from 'py:os';
+
+console.log(os.getpid()); // prints the pid from python.
+
+const list = range(0, 10); // create a range array
+console.log(len(list)); // 10
+console.log(list[2]); // 2
```

Comment:
IMO it's good to add a numpy example here.

Reply:
fixed